R/posterior_traj.R

```diff
@@ -401,7 +401,16 @@ posterior_traj <- function(object, m = 1, newdata = NULL, newdataLong = NULL,
     }
   }
     
-  newOffset <- model.offset(model.frame(terms(glmod), newX))
+  if (isTRUE(as.logical(glmod$has_offset))) {
```

Comment:
It seems that `glmod$has_offset` can be `1/0` instead of `TRUE/FALSE` for whatever reason

Reply:
Sounds good to me.